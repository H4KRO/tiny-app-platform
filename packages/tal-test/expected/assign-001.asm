main()
  entry:
    Literal           "value"
    SetLocal          name: toto
