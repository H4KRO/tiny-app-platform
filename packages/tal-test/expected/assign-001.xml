<program>
  <assign>
    <local />
    <literal />
  </assign>
</program>
